pkg/validator/fullaudit.go

```diff
@@ -1,13 +1,15 @@
 package validator
 
 import (
+	"time"
+
 	conf "github.com/reactiveops/polaris/pkg/config"
 	"github.com/reactiveops/polaris/pkg/kube"
 )
 
 const (
 	// PolarisOutputVersion is the version of the current output structure
-	PolarisOutputVersion = "0.0"
+	PolarisOutputVersion = "0.1"
```

Comment:
Will this versioning conflict be an issue? https://github.com/reactiveops/polaris/pull/127/files#diff-d344df2f44128868fd3b47e742d46d93R10 

Another PR is already planning to bump this from 0.0 to 0.1 (but the OutpufVersion will technically be different after this PR merges)

Reply:
Yup, good eyes!